### PR TITLE
VACMS-8735: Update all behat tests for new formatting/ordering.

### LINKED ISSUE #8735: Spec 3.0 behat test queries need sorts applied
## Description
Currently the spectool tests that are coming out of airtable are sorted based on last added rows.    The effect of this is all test items are going to jump around and look like massive changes on the first round of commit to a given behat feature.   It also increases the likelihood of collision and clobbers due to all updates happening at the bottom of the file.

To reduce collision management it would be helpful to add sorts to each of the queries.
https://community.airtable.com/t/sort-a-table-within-a-script/29927

## Acceptance Criteria
-  Spec tool test output matches or improves upon existing sort of behat feature rows and that they are alpha by first differentiating column and then by the next appropriate column.
- [x] access_control  sort by `Machine name`
- [x]  content_model_alerts_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_benefits_hubs_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_bundles sort by `type` and `Machine name`
- [x] content_model_campaign_landing_page_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_custom_block_fields sort by `Bundle` and `Machine name`
- [x] content_model_lc_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_media_fields sort by `Bundle` and `Machine name`
- [x] content_model_nca_facility_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_outreach_hub_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_paragraph_fields sort by `Bundle` and `Machine name`
- [x] content_model_user_guides_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_va_common_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_va_forms_content_type_fields sort by `Bundle` and `Machine name`
- [ ]content_model_vamc_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_vba_facility_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_vet_center_content_type_fields sort by `Bundle` and `Machine name`
- [x] content_model_vocabulary_fields  sort by `Bundle` and `Machine name`
- [x] media  `Machine name for the first, and `Image style` for the second.
- [x]  menus  sort by Name
- [x] views sort by Name
- [x] workflow   sort by `Machine name`

- [x] All tests are exported and committed as part of this so that there is only one commit that changes any of the existing mis-sorts so they are all resolved all at once rather than creating booby traps for the next developer to export them.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Sitewide program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide crew ` (leave Sitewide unchecked and check the specific team instead)
  - [x] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Public websites`
  - [ ] `⭐️ Facilities`
  - [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by msbtterswrth**
## Description
All behat tests in the spec tool are now ordered by the behat data view according to the ACs listed below.

Closes #8735. 

## Testing done
behat testing

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As a sitewide dev, 
1. run the behat tests
   - [ ] Validate that all tests pass
2. Then validate Acceptance Criteria from issue
   - [x] access_control  sort by `Machine name`
   - [x]  content_model_alerts_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_benefits_hubs_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_bundles sort by `type` and `Machine name`
   - [x] content_model_campaign_landing_page_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_custom_block_fields sort by `Bundle` and `Machine name`
   - [x] content_model_lc_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_media_fields sort by `Bundle` and `Machine name`
   - [x] content_model_nca_facility_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_outreach_hub_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_paragraph_fields sort by `Bundle` and `Machine name`
   - [x] content_model_user_guides_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_va_common_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_va_forms_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_vamc_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_vba_facility_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_vet_center_content_type_fields sort by `Bundle` and `Machine name`
   - [x] content_model_vocabulary_fields  sort by `Bundle` and `Machine name`
   - [x] media  `Machine name for the first, and `Image style` for the second.
   - [x]  menus  sort by Name
   - [x] views sort by Name
   - [x] workflow sort by `Machine name`
   - [x] All tests are exported and committed as part of this so that there is only one commit that changes any of the existing mis-sorts so they are all resolved all at once rather than creating booby traps for the next developer to export them.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Sitewide program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide crew ` (leave Sitewide unchecked and check the specific team instead)
  - [x] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Public websites`
  - [ ] `⭐️ Facilities`
  - [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: tests/behat/drupal-spec-tool/content_model_vba_facility_content_type_fields.feature [hunk 1]
```diff
@@ -8,7 +8,16 @@ Feature: Content model: VBA facility Content Type fields
      Scenario: Fields
        Then exactly the following fields should exist for bundles "vba_facility" of entity type node
        | Type | Bundle | Field label | Machine name | Field type | Required | Cardinality | Form widget | Translatable |
+| Content type | VBA Facility | Hours additional info | field_additional_hours_info | Text (plain) |  | 1 | Textfield |  |
+| Content type | VBA Facility | Address | field_address | Address | Required | 1 | Address |  |
+| Content type | VBA Facility | Section | field_administration | Entity reference | Required | 1 | Select list | Translatable |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_vba_facility_content_type_fields.feature:13]: This one seems odd.   There are only additions.    ??
If this is just a sorting fix the number of additions and deletions should match.
image

Comment (swirtSJW) [reply tests/behat/drupal-spec-tool/content_model_vba_facility_content_type_fields.feature:13]: It looks like there are items in here ^^ and a few others that had a "Implementation status" from the original spec of either "D" to be deleted or "W" wait to implement which were excluded from the behat test output.
If we are not going to utilize "implementation status" in the new spec, we'll have to remove anything that had those values in the old.
image

Comment (swirtSJW) [reply tests/behat/drupal-spec-tool/content_model_vba_facility_content_type_fields.feature:13]: We didn't use that status much... but it does let you stage stuff, without the requirement of having to add it the same day you implement it.

**Comment by msbtterswrth** [comment]
@swirtSJW i think i've ironed out most of the kinks. I left in the 'bad' spec for the VHA tax term change though. Should I take it out?

**Review by swirtSJW — APPROVED** [review]
Nice work on this.  Thanks for taking it on.
Behat passes.  The other failures are unrelated.

**Comment by swirtSJW** [comment]
image

**Pull request merged**